src/components/AutoCompleteInput/AutoCompleteInput.js

```diff
@@ -4,7 +4,6 @@ import styled, { css } from 'react-emotion';
 import Downshift from 'downshift';
 import { includes } from 'lodash/fp';
 
-import { zIndex } from '../../themes/default';
```

Comment:
@elanoism You shouldn't import the theme directly. All styled components have access to a `theme` prop and if you need access to the theme inside a functional component, you can use the `withTheme` HOC from `emotion-theming`. 

Sorry I didn't point this out in previous code reviews 🙈

Reply:
A question though - does this really belong inside the theme? or just insides a constants directory? Why should someone customize this inside the theme? 🤔